README.md

```diff
@@ -127,6 +128,19 @@ Are you sure? y
 Assuming MySQL is running locally, the following command runs all of the unit
 tests for the code, and should complete successfully:
 
+### Presubmit Checks
+
+Before checking in code, please ensure that the presubmit checks pass. 
+
+```
+go get -u github.com/golang/lint/golint
+go get -u github.com/kisielk/errcheck
+go get -u github.com/fzipp/gocyclo
+go get -u github.com/gordonklaus/ineffassign
+go get -u github.com/client9/misspell/cmd/misspell
+./presubmit.sh
+```
+
 ```console
```

Comment:
Should this be in the "Unit Tests" section?

(existing code nits) Btw, is "console" valid in this context? Should it be "sh" or "bash" instead? If yes, please fix throughout the file.